### PR TITLE
fix 239

### PR DESCRIPTION
**Pull request by romeokienzler**

### CODE REVIEW

File: terratorch/datamodules/__init__.py [hunk 1]
```diff
@@ -71,3 +76,6 @@
     "PASTISDataModule",
     "Sen4AgriNetDataModule"
 )
+
+if wxc_present:
+    __all__.__add__("Merra2DownscaleNonGeoDataModule")
```
Comment (fmartiescofet) [inline terratorch/datamodules/__init__.py:81]:
```suggestion
    __all__.__add__(("Merra2DownscaleNonGeoDataModule", ))
```

File: terratorch/tasks/__init__.py [hunk 2]
```diff
@@ -2,13 +2,21 @@
...
     "SemanticSegmentationTask",
     "PixelwiseRegressionTask",
     "ClassificationTask",
     "MultiLabelClassificationTask"
     "BATCH_IDX_FOR_VALIDATION_PLOTTING",
-    "WxCDownscalingTask",
 )
+
+if wxc_present:
+    __all__.__add__("WxCDownscalingTask")
```
Comment (fmartiescofet) [inline terratorch/tasks/__init__.py:22]:
```suggestion
    __all__.__add__(("WxCDownscalingTask", ))
```

**Comment by Joao-L-S-Almeida** [comment]
I believe this modifications are necessary:
```
❯ git diff
diff --git a/terratorch/datamodules/__init__.py b/terratorch/datamodules/__init__.py
index dc2df47..22d3e49 100644
--- a/terratorch/datamodules/__init__.py
+++ b/terratorch/datamodules/__init__.py
@@ -78,4 +78,4 @@ __all__ = (
 )
 
 if wxc_present:
-    __all__.__add__("Merra2DownscaleNonGeoDataModule")
\ No newline at end of file
+    __all__.__add__(("Merra2DownscaleNonGeoDataModule",))
diff --git a/terratorch/tasks/__init__.py b/terratorch/tasks/__init__.py
index 9dfc132..7c26282 100644
--- a/terratorch/tasks/__init__.py
+++ b/terratorch/tasks/__init__.py
@@ -19,4 +19,4 @@ __all__ = (
 )
 
 if wxc_present:
-    __all__.__add__("WxCDownscalingTask")
\ No newline at end of file
+    __all__.__add__(("WxCDownscalingTask",))

```

**Comment by romeokienzler** [comment]
@fmartiescofet @Joao-L-S-Almeida I've just trusted you on this one and merged :)

any insights why this is necessary?

**Comment by fmartiescofet** [comment]
@romeokienzler A string can't be added with a tuple, you need to add a second tuple, `("WxCDownscalingTask",)` creates a tuple with one element that can be added to the original tuple.

**Pull request merged**